CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -1,4 +1,4 @@
-﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
 
 namespace CompanyEmployees.Core.Domain.Entities
```

Comment:
The newly added `Salary` property should be positioned after `CompanyId` for consistency and correct domain modeling. Additionally, it should ideally be initialized or validated for business correctness (e.g., non-negative).

```suggestion
public Guid CompanyId { get; set; }
public decimal Salary { get; set; }
```